lib/galaxy/config/__init__.py

```diff
@@ -531,7 +531,11 @@ def __init__(self, **kwargs):
         self._process_config(kwargs)
 
     def _load_schema(self):
-        return AppSchema(GALAXY_CONFIG_SCHEMA_PATH, GALAXY_APP_NAME)
+        # Schemas are symlinked to the root of the galaxy-app package
+        config_schema_path = os.path.join(os.path.dirname(__file__), os.pardir, 'config_schema.yml')
```

Comment:
This should be fine. However, I wonder, would it make more sense to check the `GALAXY_CONFIG_SCHEMA_PATH` location first? I assume this path - `lib/galaxy/webapps/galaxy/config_schema.yml` will *not* exist in this context? (if it will exist - then none of this is relevant). If so, it is unlikely, but not impossible to have a copy of `config_schema.yml` inside `lib/galaxy/` - and then this stray file would be loaded instead - and that would lead to some ugly bugs if the file were outdated/edited/etc.

Reply:
Yeah that path will not exist in the installed context (unless you happened to call `galaxy-main` from the root of a Galaxy source tree). We actually did a fair amount of this sort of stuff in #921 but I would have to go back and check whether we check the source tree path first or the installed path first in most cases.

Reply:
So, you think switching them is unnecessary? If so, that's fine by me.

Reply:
Actually I forgot to follow up on this, but I'll dig deeper into it for the next PR.